selfdrive/controls/controlsd.py

```diff
@@ -192,6 +193,10 @@ def update_events(self, CS):
       self.events.add(EventName.controlsInitializing)
       return
 
+    # Disable on rising edge of gas or brake. Also disable on brake when speed > 0
+    if (CS.gasPressed and self.active) or (CS.brakePressed and not CS.standstill):
```

Comment:
this isn't the same as before. now, we don't disengage on the rising edge of brake